packages/time-picker/src/vaadin-time-picker.js

```diff
@@ -298,11 +301,23 @@ class TimePicker extends PatternMixin(InputControlMixin(ThemableMixin(ElementMix
 
       /** @private */
       _inputContainer: Object,
+
+      /** @private */
+      __minTime: {
+        type: Object,
+        computed: '__computeMinTime(i18n, min)',
+      },
+
+      /** @private */
+      __maxTime: {
+        type: Object,
+        computed: '__computeMaxTime(i18n, max)',
+      },
```

Comment:
Could probably avoid the additional computed properties (more state), and internally fall back to the default values `min || MIN_ALLOWED_TIME`?

Reply:
This is also an option. An advantage of using computed properties is that we can avoid parsing min and max all around, but it is more just a nice addition. I don't mind doing as you suggested.

Reply:
Done.